src/RandomDataGenerator/FieldOptions/FieldOptionsGuid.cs

```diff
@@ -4,5 +4,7 @@ namespace RandomDataGenerator.FieldOptions
     public class FieldOptionsGuid : FieldOptionsAbstract, IFieldOptionsGuid
     {
         public bool Uppercase { get; set; } = true;
+
+        public string Format { get; set; } = "D";
```

Comment:
Can you please also update the GUI with this new Format field (a dropdownlist with the options defined like https://docs.microsoft.com/en-us/dotnet/api/system.guid.tostring?view=netframework-4.8).

Reply:
Updated. Please review